django/contrib/auth/views.py

```diff
@@ -425,6 +425,9 @@ def form_valid(self, form):
         return super(PasswordResetView, self).form_valid(form)
 
 
+INTERNAL_RESET_TOKEN = 'set-password'
```

Comment:
I'd split up the URL name and the session name, especially I'd use something more descriptive ala `_password_reset_token` as session name.